spec/dummy/config/boot.rb

```diff
@@ -1,3 +1,3 @@
-ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../../../Gemfile", __dir__)
+ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
```

Comment:
This needs to be reverted because the `Gemfile` is in the root of the gem and not in the root of the `spec/dummy` app.

Reply:
@tvdeyen Updated